test/Swashbuckle.AspNetCore.SwaggerGen.Test/ConfigureSwaggerGeneratorOptionsTests.cs

```diff
@@ -15,4 +21,27 @@ public static void DeepCopy_Copies_All_Properties()
         // to SwaggerGeneratorOptions and ConfigureSwaggerGeneratorOptions.DeepCopy() needs to be updated
         Assert.Equal(18, publicProperties.Length);
     }
+
+    [Fact]
+    public static void AddingDocumentFilterInstance_WhenConfiguringOption_SameInstanceIsAdded()
+    {
+        var webhostingEnvironment = Substitute.For<IWebHostEnvironment>();
```

Comment:
@martincostello To help with testing, I added NSubstitute, to be able to mock some needed input, is that OK? Or should I use another mocking library? Or create a manual mock of IWebHostEnvironment?

(I don't want to write all tests to only then discover I need to change)

Reply:
Adding NSub is fine.